### PR TITLE
DFAST-159: add a case description

### PR DESCRIPTION
**Pull request by ralph-peelen-deltares**

### CODE REVIEW

File: dfastmi/gui/dialog_view.py [hunk 1]
```diff
@@ -748,7 +780,7 @@ def _menu_load_configuration(self) -> None:
             caption=gui_text("select_cfg_file"), filter="Config Files (*.cfg)"
         )
         filename = fil[0]
-        if filename != "" and not self._view_model.load_configuration(filename):
+        if filename != "" and not self._view_model.load_configu_upration(filename):
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:783]: Fix name

File: dfastmi/messages.UK.ini [hunk 2]
```diff
@@ -388,4 +388,8 @@ interest area clipped to the range {low} to {high} km
 [legacy_river_config_loaded]
 The configuration for rivieren is legacy. If you still want to use the GUI use an older version of the application.
 [gui_condition_group_name]
-Output of Reference Simulations
\ No newline at end of file
+Output of Reference Simulations
+[gui_case_description]
+Case description
+[gui_case_description_tooltip]
+What is the name of the de case?
```
Comment (priscavdsluis) [inline dfastmi/messages.UK.ini:395]:
```suggestion
What is the name of the case?
```

File: dfastmi/gui/dialog_view.py [hunk 3]
```diff
@@ -614,7 +619,34 @@ def _create_branch_input(self, layout: QFormLayout) -> None:
...
+        self._case_description.editingFinished.connect(self._update_case_description)        
+        self._case_description.setToolTip(gui_text("case_description_tooltip"))
+        case_description_label = QLabel(gui_text("case_description"), self._win)
+        layout.addRow(case_description_label, self._case_description) 
+
+    def _update_case_description(self) -> None:
+        """
+        Update case description.
+
+        Returns:
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:644]: FYI, if a function returns None, it is better to not document this.

Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_view.py:644]: Good to know!

File: dfastmi/gui/dialog_view.py (hunk 1 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:783]: Yes, thanks. Updated

File: dfastmi/gui/dialog_view.py (hunk 3 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:644]: Will remove here and keep in mind for future. Will not update rest of code.

**Pull request merged**